.flowconfig

```diff
@@ -19,6 +19,7 @@
 ; "node_modules/react-native" but in the source repo it is in the root
 .*/Libraries/react-native/React.js
 .*/Libraries/react-native/ReactNative.js
+.*/Libraries/Renderer/src/renderers/shared/utils/ReactErrorUtils.js
```

Comment:
I've also noticed this recently, when I checked it came from react-dom that we install as a dev dependency (I think it's needed for some test utils to work). If it is the case we should just ignore react-dom instead of our module.

```
.*/node_modules/react-dom/.*
```